benchmark/benches/benchmark.rs

```diff
@@ -89,6 +92,7 @@ pub struct NetPacketWrite<G> {
 }
 
 #[derive(Debug, Deserialize)]
+#[cfg_attr(feature = "speedy", derive(speedy::Readable))]
```

Comment:
Not sure `speedy` can deserialize this properly. How is it even compiled?
You're not using this type below for benching `speedy` though.

Reply:
Ah, sorry, yes, you're right. I just went trigger-happy with adding the `#[derive]`s.

Reply:
I tried to use `NetPacketRead` with speedy and it end up not compiling.
Seems that this derive macro didn't actually implement `Readable` for `NetPackedRead`. Apparently due to where clause like `Lazy<[G]>: Readable<...>`

Reply:
Alkahest does the same :)
A bound is added for all field types that reference at least one generic parameter.